src/components/Section/History/Employment/EmploymentItem.test.jsx

```diff
@@ -133,6 +133,42 @@ describe('The employment component', () => {
     expect(component.find('.reason-description').length).toBe(0)
   })
 
+  it('does not display reason for leaving for unemployment', () => {
+    const past = daysAgo(today, 365 * 3)
```

Comment:
The setups for these two tests are identical, so we should move them into a `beforeEach` block.

If possible, I think it would be cleaner to `find` the components you expect to to not be there by their names, rather than the css selectors (`'Reprimand'` and `'ReasonLeft'`).

Reply:
Makes sense, made some tweaks: https://github.com/18F/e-QIP-prototype/pull/1079/commits/389bff2c584dc1076b98112163c5e4980356ec52